Source/WebCore/bindings/js/JSDOMConvertBoolean.h

```diff
@@ -27,16 +27,24 @@
 
 #include "IDLTypes.h"
 #include "JSDOMConvertBase.h"
+#include <JavaScriptCore/JSGlobalObject.h>
 
 namespace WebCore {
 
 template<> struct Converter<IDLBoolean> : DefaultConverter<IDLBoolean> {
-
     static constexpr bool conversionHasSideEffects = false;
+    using Result = ConversionResult<IDLBoolean>;
 
-    static bool convert(JSC::JSGlobalObject& lexicalGlobalObject, JSC::JSValue value)
+    static ConversionResult<IDLBoolean> convert(JSC::JSGlobalObject& lexicalGlobalObject, JSC::JSValue value)
```

Comment:
Maybe we can just use Result here since we're defining using?

Reply:
Good catch. That was the intent. Fixed.